### PR TITLE
MathGame submission

### PR DESCRIPTION
**Pull request by CBoton**

### CODE REVIEW

**Comment by stevietv** [comment]
Hi @CBoton - thanks for your first project submission! I've taken a look over it and for now it needs a bit of work. I'm missing the MathGame.Models folder where your definition of Game is stored. Can you take a look?

One additional feedback: 
Make sure to submit your next projects within a subfolder, like `MathGame.CBoton` instead of files directly in the top directory.

**Comment by CBoton** [comment]
Hi Stevie,

I have added the Models folder and Game.Cs file.

I will make sure to add the entire folder next time.

Thanks,
Curtis

**Comment by stevietv** [comment]
hi @CBoton - your models folder currently has a trailing space:
image

this makes it impossible to checkout in windows as its an invalid path:
image


can you fix that path and re-push?

File: GameEngine.cs [hunk 1]
```diff
@@ -0,0 +1,334 @@
+﻿
+
+using MathGame.Models;
+using System;
+using System.Diagnostics;
+using static System.Formats.Asn1.AsnWriter;
```
Comment (stevietv) [inline GameEngine.cs:6]: unnecessary using - this library isn't being used

File: GameEngine.cs [hunk 2]
```diff
@@ -0,0 +1,334 @@
+﻿
+
+using MathGame.Models;
+using System;
```
Comment (stevietv) [inline GameEngine.cs:4]: unnecessary using - this is implicitly used, no need to add.

File: Program.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿
+using MathGame;
+using Microsoft.VisualBasic;
```
Comment (stevietv) [inline Program.cs:3]: unnecessary using - this library isn't being used

**Review by stevietv — CHANGES_REQUESTED** [review]

**Comment by stevietv** [comment]
Hi @CBoton - great work, the progam does indeed what it needs to do, thanks for the submission!
In order to mark it as approved there are few tweaks to make to keep with the required code conventions.

**Comment by CBoton** [comment]
Hi Stevie,

I removed the using statements.

Thanks

**Comment by stevietv** [comment]
you can't remove all 4 from GameEngine.cs - only the two I commented.
Its important to test your code once you edit it, right now the program won't run again.

**Comment by CBoton** [comment]
Sorry Stevie, I'm on the road and just updated from my phone on GitHub so
couldn't test.

**Comment by stevietv** [comment]
No worries, update it properly once youre around again and then we'll approve it :)

**Comment by CBoton** [comment]
Hi Stevie,

I think it's good now.

Thanks,
Curtis

**Comment by stevietv** [comment]
thanks for working with me to finalise it! its all approved now! ✅

**Pull request closed without merging**